integrations/client/test_delphi_epidata.py

```diff
@@ -236,32 +244,32 @@ def fetch(geo):
         **self.params_from_row(rows[0], geo_value=geo)
       )
 
+    self.maxDiff = None
     # test fetch all
     request = fetch('*')
     self.assertEqual(request['message'], 'success')
     self.assertEqual(request['epidata'], counties)
     # test fetch a specific region
-    request = fetch('11111')
+    request = fetch('{}'.format(fips[0]))
     self.assertEqual(request['message'], 'success')
-    self.assertEqual(request['epidata'], [counties[1]])
+    self.assertEqual(request['epidata'], [counties[0]])
     # test fetch a specific yet not existing region
     request = fetch('55555')
-    self.assertEqual(request['message'], 'no results')
+    self.assertEqual(request['message'], 'invalid geo_value for the requested geo_type')
     # test fetch a multiple regions
-    request = fetch(['11111', '22222'])
+    request = fetch(['{}'.format(fips[0]), '{}'.format(fips[1])])
     self.assertEqual(request['message'], 'success')
-    self.assertEqual(request['epidata'], [counties[1], counties[2]])
+    self.assertEqual(request['epidata'], [counties[0], counties[1]])
     # test fetch a multiple regions in another variant
-    request = fetch(['00000', '22222'])
+    request = fetch(['{}'.format(fips[0]), '{}'.format(fips[2])])
     self.assertEqual(request['message'], 'success')
     self.assertEqual(request['epidata'], [counties[0], counties[2]])
     # test fetch a multiple regions but one is not existing
-    request = fetch(['11111', '55555'])
-    self.assertEqual(request['message'], 'success')
-    self.assertEqual(request['epidata'], [counties[1]])
+    request = fetch(['{}'.format(fips[0]), '55555'])
+    self.assertEqual(request['message'], 'invalid geo_value for the requested geo_type')
     # test fetch a multiple regions but specify no region
     request = fetch([])
-    self.assertEqual(request['message'], 'no results')
+    self.assertEqual(request['message'], 'invalid geo_value for the requested geo_type')
```

Comment:
it would be good to have a test for a return of `'no results'`...  maybe add one more item to your `fips` list and do a `request = fetch(fips[3])`?